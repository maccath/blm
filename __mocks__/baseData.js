const baseData = {
  site: {
    siteMetadata: {
      author: "Site Author",
      description: "Site description",
      title: "Site Title",
    },
  },
  placeholderImage: {
    childImageSharp: {
      fluid: {
        src: "icon.png",
        srcSet: "icon.png 50w, icon.png 100w",
        sizes: "(max-width: 200px) 100vw, 200px",
        base64: "data:image/png;base64,iVBORw0KG",
        aspectRatio: 1,
      },
    },
  },
}

export default baseData
