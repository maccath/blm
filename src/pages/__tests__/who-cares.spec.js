import React from "react"
import renderer from "react-test-renderer"
import { useStaticQuery } from "gatsby"
import "jest-styled-components"
import baseData from "../../../__mocks__/baseData"

import WhoCares from "../who-cares"

beforeEach(() => {
  useStaticQuery.mockImplementation(() => baseData)
})

describe("Who Cares? page", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<WhoCares />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
