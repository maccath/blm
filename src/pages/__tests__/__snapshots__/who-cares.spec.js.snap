// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Who Cares? page renders correctly 1`] = `
<div
  style={
    Object {
      "margin": "0 auto",
      "maxWidth": 960,
      "padding": "0 1.0875rem 1.45rem",
    }
  }
>
  <main>
    <h1>
      Why should I take it from you?
    </h1>
    <p>
      Fair point.
    </p>
    <p>
      Would you take it from the ice cream manufacturer 
      <a
        href="https://www.benjerry.com/about-us/media-center/dismantle-white-supremacy"
      >
        Ben And Jerry's
      </a>
      ?
    </p>
    <div>
      <blockquote
        className="twitter-tweet"
      >
        <p
          dir="ltr"
          lang="en"
        >
          The murder of George Floyd was the result of inhumane police brutality that is perpetuated by a culture of white supremacy.
           
          <a
            href="https://t.co/YppGJKHkyN"
          >
            https://t.co/YppGJKHkyN
          </a>
           
          <a
            href="https://t.co/YABzgQMf69"
          >
            pic.twitter.com/YABzgQMf69
          </a>
        </p>
        — Ben & Jerry's (@benandjerrys)
         
        <a
          href="https://twitter.com/benandjerrys/status/1267875868673794048?ref_src=twsrc%5Etfw"
        >
          June 2, 2020
        </a>
      </blockquote>
      <script
        async={true}
        charSet="utf-8"
        src="https://platform.twitter.com/widgets.js"
      />
    </div>
    <p>
      How about popular comedian 
      <a
        href="https://www.youtube.com/watch?v=qUZIzYwmEDE"
      >
        James Corden
      </a>
      ?
    </p>
    <div>
      <iframe
        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
        allowFullScreen=""
        frameBorder="0"
        height="315"
        src="https://www.youtube.com/embed/qUZIzYwmEDE"
        title="James Corden: It's Time for Change in the US"
        width="560"
      />
    </div>
    <p>
      Netflix?
    </p>
    <div>
      <blockquote
        className="twitter-tweet"
      >
        <p
          dir="ltr"
          lang="en"
        >
          To be silent is to be complicit. 
          <br />
          Black lives matter. 
          <br />
          <br />
          We have a platform, and we have a duty to our Black members, employees, creators and talent to speak up.
        </p>
        — Netflix (@netflix)
         
        <a
          href="https://twitter.com/netflix/status/1266829242353893376?ref_src=twsrc%5Etfw"
        >
          May 30, 2020
        </a>
      </blockquote>
      <script
        async={true}
        charSet="utf-8"
        src="https://platform.twitter.com/widgets.js"
      />
    </div>
    <p>
      Or the company I work with, 
      <a
        href="https://www.skillshare.com/blog/company/a-note-from-our-ceo-and-commitment-to-anti-racism?via=site-banner"
      >
        Skillshare
      </a>
      ?
    </p>
    <div
      className="mt-6 flex space-x-4"
    >
      <a
        class="flex-initial rounded-lg bg-white text-purple p-3 no-underline"
        href="/what-to-do"
      >
        Yeah, OK, but what can I do about it?
      </a>
    </div>
  </main>
  <footer
    className="css-1bxtzlt"
  >
    <p>
      Compiled by 
      <a
        href="http://katy-ereira.co.uk"
      >
        Katy Ereira
      </a>
       /
       
      <a
        href="http://twitter.com/maccath"
      >
        @maccath
      </a>
       | Fist icon by
       
      <a
        href="https://thenounproject.com/cesarreynoso562/"
      >
        Cesar Reynoso
      </a>
    </p>
  </footer>
</div>
`;
